### PR TITLE
RATEST-267: Removed SauceLabs implementation from TestBase Class

### PR DESCRIPTION
**Pull request by Muta-Jonathan**
## Issue I worked on

👀 https://issues.openmrs.org/browse/RATEST-267

## 
#### Thank You :)

### CODE REVIEW

**Comment by Muta-Jonathan** [comment]
cc @sherrif10 @kdaud @dkayiwa

**Comment by kdaud** [comment]
@sherrif10 don't we have a future plan with Sauce labs?

**Comment by Muta-Jonathan** [comment]
**Firefox failing** due to master

**Comment by sherrif10** [comment]
@kdaud  source labs have been left out of use for 7 years. we don't have any plan for future maintenance.

**Comment by kdaud** [comment]
> we don't have any plan for future maintenance.

so do you approve the PR?

**Comment by sherrif10** [comment]
@Muta-Jonathan  we have a serious bug that kdaud is working currently so I guess we  look into this when things settle.

**Comment by Muta-Jonathan** [comment]
@sherrif10 Yeah no prob

**Comment by kdaud** [comment]
Update the branch with changes from master.

**Comment by sherrif10** [comment]
Any response to the above comment @Muta-Jonathan

**Comment by Muta-Jonathan** [comment]
@sherrif10 doesn't it call for re-running the tests please if so trigger them for me

**Comment by sherrif10** [comment]
> doesn't it call for re-running the tests please if so trigger them for me

Is your branch up to date with master branch

**Comment by Muta-Jonathan** [comment]
yeah I see no merge conflicts

**Comment by sherrif10** [comment]
Have done the needfull.

**Comment by sherrif10** [comment]
Please make sure this line https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/src/test/resources/features/refapp-2.x/015-reports/reports.feature#L17 is commented out as it is on master branch.

**Comment by Muta-Jonathan** [comment]
@sherrif10 should I do it

**Comment by sherrif10** [comment]
> should I do it

You can update your branch to the latest changes.

**Review by sherrif10 — APPROVED** [review]
Thanks @Muta-Jonathan .  cc @kdaud

File: qaframework-bdd-tests/src/test/resources/features/refapp-2.x/015-reports/reports.feature [hunk 1]
```diff
@@ -14,4 +14,4 @@ Feature: Reports Management
     And user enters start date
     And user enters end date
     And user clicks on request report button
-    Then user clicks on view report link
+    #Then user clicks on view report link
```
Comment (kdaud) [inline qaframework-bdd-tests/src/test/resources/features/refapp-2.x/015-reports/reports.feature:17]: @Muta-Jonathan is this change part of the ticket requirement?

Comment (Muta-Jonathan) [reply qaframework-bdd-tests/src/test/resources/features/refapp-2.x/015-reports/reports.feature:17]: @kdaud according to https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/src/test/resources/features/refapp-2.x/015-reports/reports.feature ...its commented out on master....and my PR is failing to update as on master

Comment (kdaud) [reply qaframework-bdd-tests/src/test/resources/features/refapp-2.x/015-reports/reports.feature:17]: This makes the change being reflected as part of the ticket requirement which false.

>  my PR is failing to update as on master

do you find this link helpful? https://gist.github.com/whoisryosuke/36b3b41e738394170b9a7c230665e6b9

**Comment by Muta-Jonathan** [comment]
am gonna close dis due to the mess I created in this pr

**Comment by Muta-Jonathan** [comment]
opened new here @kdaud @sherrif10 https://github.com/openmrs/openmrs-contrib-qaframework/pull/365

**Pull request closed without merging**